queries/sql/indents.scm

```diff
@@ -0,0 +1,26 @@
+[
+ (select_expression)
```

Comment:
This should be select

Reply:
Are you sure? This is the tree for that statement:
```
statement [1, 0] - [4, 9]
  select [1, 0] - [3, 5]
    keyword_select [1, 0] - [1, 6]
    select_expression [2, 0] - [3, 5]
      term [2, 0] - [2, 1]
        value: field [2, 0] - [2, 1]
          name: identifier [2, 0] - [2, 1]
      term [3, 4] - [3, 5]
        value: field [3, 4] - [3, 5]
          name: identifier [3, 4] - [3, 5]
  from [4, 0] - [4, 8]
    keyword_from [4, 0] - [4, 4]
    relation [4, 5] - [4, 8]
      table_reference [4, 5] - [4, 8]
        name: identifier [4, 5] - [4, 8]
```

`keyword_select` is a child of `select` and should not be indented.